src/st/st0/2C564.c

```diff
@@ -42,52 +46,52 @@ static u8 D_80180A20[] = {
 static u8 D_80180A2C[] = {0x07, 0x47, 0x08, 0x48, 0x07, 0x49, 0x08, 0x4A, 0x1A,
                           0x4B, 0x07, 0x4C, 0x05, 0x4D, 0x28, 0x4E, 0xFF, 0x00};
 static u8 D_80180A40[] = {0x02, 0x5A, 0x02, 0x5B, 0xFF, 0x00};
-static u16 D_80180A48[] = {0x20, 0x40, 0x60, 0x80, 0xA0, 0xC0, 0xE0, 0x90};
+static s16 D_80180A48[] = {0x20, 0x40, 0x60, 0x80, 0xA0, 0xC0, 0xE0, 0x90};
 static Point16 D_80180A58[] = {{16, 28}, {20, 60}};
-static Point16 D_80180A60[] = {
-    {12, 12}, {12, -12}, {-12, 12}, {-12, -12}, {80, 64}, {48, 34},
-    {26, 20}, {18, 16},  {15, 13},  {13, 12},   {12, 11}, {11, 10},
-    {10, 10}, {9, 9},    {9, 8},    {8, 8},     {8, 4}};
+static Point16 D_80180A60[] = {{12, 12}, {12, -12}, {-12, 12}, {-12, -12}};
+// Possible this is an unused u8 animation from the next file.
+// It is stripped on PSP
+static Point16 unused[] = {
+    {80, 64}, {48, 34}, {26, 20}, {18, 16}, {15, 13}, {13, 12}, {12, 11},
+    {11, 10}, {10, 10}, {9, 9},   {9, 8},   {8, 8},   {8, 4}};
```

Comment:
This is stripped on PSP, and as such isn't part of `D_80180A60`. Given the next lot of data is animations from `2DAC8`, I suspect this is likely an unused animation but will need to dig into that further to confirm.